Intersect.Server.Core/Entities/Player.cs

```diff
@@ -7172,6 +7189,17 @@ public bool TryChangeName(string newName)
 
         }
 
+        public bool TryAddToInstanceController()
+        {
+            if (InstanceProcessor.TryGetInstanceController(MapInstanceId, out var newInstController))
```

Comment:
Why is it called `new`InstController? Since TryGetInstanceController() doesn't create controllers I am wondering if the variable is just misnamed or if the code should be different.

Reply:
fixed name